WordPress/Classes/ViewRelated/Site Creation/Wizard/SiteCreationWizardStepOrderer.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import AutomatticTracks
+
+struct SiteCreationWizardStepOrderer {
```

Comment:
The inspiration behind this was when I started to update the `addressStep` to figure out if it should or shouldn't be shown, and found myself juggling Site Name being mutually exclusive of Site Address, but depending on Site Intent.